model/layout.md

```diff
@@ -3,17 +3,22 @@
 
 <a name="Layout"></a>
 * [Layout](#Layout)
-    * [.direction](#Layout+direction) : <code>String</code>
-    * [.padding](#Layout+padding) : <code>Object</code>
-    * [.gap](#Layout+gap) : <code>Number</code>
-    * [.sizingMode](#Layout+sizingMode) : <code>String</code>
-    * [.alignment](#Layout+alignment) : <code>String</code>
+    * _instance_
+        * [.direction](#Layout+direction) : [<code>DIRECTION</code>](#Fill+DIRECTION)
```

Comment:
~Fill~ Layout. 

Also, headline anchors are not correct. Check the following file for the right URL format.